### ISSUE TITLE
Empty zip code list rendering with invalid URL

### ISSUE THREAD

**Issue by lucylee-412** [lucylee-412/city-api#9]
Currently, the zip codes list is set up to always render, and be populated upon a successful entry.
However, this means an empty zip code list shows up with the error as well.
Ideally, the div container for the zip codes should only display after a valid city is submitted.

**Comment by lucylee-412** [comment]
We will need to use conditional rendering, but I am strugging to get it to work within the App component. My output is being rendered as plain text instead of HTML.

I am considering making a new component.

**Comment by lucylee-412** [comment]
Got it to work. The way I wrote up my conditionals was correct, but I'd accidentally deleted my zipcodes declaration, and I'd also mistakenly wrapped it all as a string (hence why it was showing up as plain text, durr).

Here is the renderZip function that will be called upon form submission.
 ```
const renderZip = () => {
            if (this.state.errorMessage !== "No results") {
                return <div className='container'>
                    <div className='container-top'>Zip Codes Associated With This City:</div>
                    <ul>
                        {zipcodes.map((zip) => (
                            <li key={zip}>{zip}</li>
                        ))}
                    </ul>
                    </div>;
            }
            else {
                return <div>{this.state.errorMessage}</div>
            }
        }
```
`{this.state.isSubmitted && renderZip()}`

**Comment by lucylee-412** [comment]
Commit 201e5a2c7f23c58e894d0c25d0540dcc419fc4a1 closes #9
